wix-media-v2/wix-media-v2/Files/guides/upload-api.md

```diff
@@ -82,6 +82,8 @@ async function uploadMyFile(uploadUrl, fileContent) {
 }
 ```
 
+## Add Uploaded Media
```

Comment:
"Add" is too vague a word here. Also, per our style, headings are in sentence case.

```suggestion
## Use uploaded media in your site
```